streaming/java/streaming-runtime/src/main/java/org/ray/streaming/runtime/core/graph/executiongraph/ExecutionEdge.java

```diff
@@ -0,0 +1,66 @@
+package org.ray.streaming.runtime.core.graph.executiongraph;
+
+import com.google.common.base.MoreObjects;
+import java.io.Serializable;
+import org.ray.streaming.api.partition.Partition;
+
+/**
+ * Edge to attach execution vertex.
+ */
+public class ExecutionEdge implements Serializable {
+
+  private final ExecutionVertex producer;
```

Comment:
I prefer `srcVertex` and `targetVertex`. `producer`/`consumer` sounds like a client